### PR TITLE
Document hiring process using Workable

### PR DESCRIPTION
**Pull request by zupo**
Refs niteoweb/operations#529

### CODE REVIEW

File: apps-we-use/workable.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Using Workable
+
+[Workable](https://www.workable.com/) is a all-in-one recruiting SaaS. We use it to source and evaluate candidates, track applicants and keep all information in one place so we can then make informed decisions on who to hire.
```
Comment (marbeann) [inline apps-we-use/workable.md]: >...is a all-in-one recruiting SaaS ->is **an** all-in-one recruiting SaaS

File: apps-we-use/workable.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+We strive to have a bureaucracy- and ego- free hiring process that provides a welcoming experience for new Niteans. It's impossible to thoroughly test someone's fit with interviews and phone calls, so we rather do a couple of basic tests and then take them on for a [trial period].
+
+That should be your mindset when conducting interviews: you are not searching for an ideal candidate. You are on the lookout for big "no-no"'s. If someone's lacking some skills, they can be learned. If someone is used to different type of tools or processes, they can be changed. Look for problem solving prowess, ability to communicate what went wrong and ownership of mistakes.
+
+### Hiring workflow
+
+Workable guides us through the following stages of hiring a new Nitean:
+
+1. Sourcing
```
Comment (dmurko) [inline apps-we-use/workable.md]: This doesn't translate in Markdown well:

screen shot 2018-05-14 at 09 33 24

File: apps-we-use/workable.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+   We post the job ads to various free and paid options. This is decided on a per-position basis, but here's the general idea:
+   * Workable posts to 15+ US-based free boards automatically (Glassdoor, Indeed, etc.)
+   * We post manually to Slovenian boards: slo-tech.com, internetmojster.com, facebook.com/groups/sluzbezadeveloperje
+   * We post to twitter.com/teamniteo and facebook.com/niteo.
+   * Sometimes we make an add campaign on Facebook or buy ads on StackOverflow.
```
Comment (marbeann) [inline apps-we-use/workable.md]: add -> ad

File: apps-we-use/workable.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+2. Application
+   Interested candidates get to our Application Form, hosted by Workable. The form needs to be short, with only general questions. Advanced (technical) questions to asses basic skills for the position will be done in the next step.
+
+   Members of the hiring team +1/-1 candidates based on application form answers. Whoever gets two or more +1s gets transitioned to the next step: Questionnaire. Use the `Advanced Questionnair - <position>` email template to send the questions/tasks to the candidate.
```
Comment (dmurko) [inline apps-we-use/workable.md]: Typo? `Advanced Questionnair - <position>`, an `e` missing.

File: apps-we-use/workable.md (hunk 2 above)
Comment (marbeann) [inline apps-we-use/workable.md]: The use of full stop is not consistent in the list.

File: apps-we-use/workable.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+3. Questionnaire
+   When the answers arrive we rate the candidate using the scorecard that is prepared in advance, and matches the `Advanced Questionnaire - <poisition>` template. Discussion happens in comments inside Workable, so everything is kept in one place.
```
Comment (dmurko) [inline apps-we-use/workable.md]: A different typo - `<poisition>`.

File: apps-we-use/workable.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+   Whoever gets two or more +1s gets transitioned to the next step: Phone Screen.
+
+4. Phone Screen
+   This is a quick chat over Skype/Hangouts/etc. with @marbeann. The goal is to confirm the candidate is able to communicate via video conferencing. Is their connection OK? Is their English OK? Can they hold a discussion on a subject?
+
+   @marbe also confirms the following:
```
Comment (dmurko) [inline apps-we-use/workable.md]: typo: marbeann

Comment (marbeann) [inline apps-we-use/workable.md]: `@marbeann` ? or Marbe

File: apps-we-use/workable.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+   * weekly commitment for the trial period (full- or part-time),
+   * when can they start,
+   * expected ballpark salary (more detailed salary discussion in next step).
+
+5. Interview
+   Before the interview we need to select the Nitean who will mentor the candidate should the candidate start the trial period.
+
+   Then the proposed mentor for the candidate and at least one of founders (partners?), along with anyone who is interested and/or who will work with the candidate, get on Zoom to have a chat with the candidate.
```
Comment (dmurko) [inline apps-we-use/workable.md]: `one of the Partners`

File: apps-we-use/workable.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+7. Hired
+   The candidate accepts our offer and we start the [onboarding process].
+
+[trial period]: https://github.com/niteoweb/handbook/blob/master/roles/onboarding.md#trial-period
+[onboarding process]: https://github.com/niteoweb/handbook/blob/master/roles/onboarding.md
+
+
+
+start time
```
Comment (dmurko) [inline apps-we-use/workable.md]: Remove?

File: apps-we-use/workable.md (hunk 7 above)
Comment (zupo) [reply apps-we-use/workable.md]: I specifically went for Founders as Partners does not mean much to people outside of Niteo. But for any candidate it always feels like one matters if one of founders is part of the interview.

**Comment by cas--** [comment]
This is a hiring process document that happens to use workable at some point. The idea for apps-we-use is that they are used daily by everyone and need specific instructions for their setup and use. 

Therefore I think this should be a document named `roles/hiring.md`

**Comment by cas--** [comment]
Also I don't see this as close to being ready for merge, too many small issues need resolving, ordinarily I would block this because it's been approved for merge...

**Comment by zupo** [comment]
I have addressed all comments (apart from Partners vs. Founders from @dmurko). 

I've run the text through Grammarly.

File: roles/hiring.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+   We post the job ads to various free and paid options, on a per-position basis, but here's a general idea:
+   * Workable posts to 15+ US-based free boards automatically (Glassdoor, Indeed).
+   * We post manually to Slovenian boards: slo-tech.com, internetmojster.com, facebook.com/groups/sluzbezadeveloperje.
+   * We post to twitter.com/teamniteo and facebook.com/niteo.
+   * Sometimes we run an ad campaign on Facebook or buy ads on StackOverflow.
+
+2. Application
+
+   Interested candidates get to our Application Form, hosted by Workable. The form needs to be short, with only general questions. Resist the temptation to ask advanced (technical) questions to asses needed skills for the position. We do these in the next step.
```
Comment (cas--) [inline roles/hiring.md]: I think we need to `assess` rather than many `asses` ;)

File: roles/hiring.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+3. Questionnaire
+
+   Members of the hiring team rate (+1/-1) candidates based on application form answers. Whoever gets two or more +1s gets transitioned to the next step: the Questionnaire. Use the `Advanced Questionnaire - <position>` email template to send the questions/tasks to the candidate.
+
+   When the answers arrive we rate the candidate using the scorecard that is prepared in advance and matches the `Advanced Questionnaire - <position>` template. Discussion happens in comments inside Workable, so everything is kept in one place.
+
+   Whoever gets two or more +1s gets transitioned to the next step: Phone Screen.
+
+4. Phone Screen
```
Comment (cas--) [inline roles/hiring.md]: Perhaps `Interview Screening`?

File: roles/hiring.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+   A chat over Skype, Hangouts or Zoom with @marbeann. Calls can last anywhere between 10 and 30. The goal is to confirm the candidate is able to communicate via video conferencing. Is their connection OK? Is their English OK? Can they hold a discussion on a subject?
```
Comment (cas--) [inline roles/hiring.md]: Does it specifically need to be `@marbeann`? I feel the document should be more general.

File: roles/hiring.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+
+   @marbeann also confirms the following:
+   * Weekly commitment for the trial period (full- or part-time).
+   * When can they start?
+
+5. Interview
+
+   Before the interview we need to select the Nitean who will mentor the candidate should the candidate start the trial period.
+
+   Then the proposed mentor for the candidate and at least one of founders, along with anyone who is interested or will work with the candidate, get on Zoom to have a chat with the candidate.
```
Comment (cas--) [inline roles/hiring.md]: `get on Zoom` sounds a bit off, perhaps `create a meeting room on Zoom`

File: roles/hiring.md (hunk 10 above)
Comment (zupo) [reply roles/hiring.md]: Interview is the next step, I don't want to cause confusion. Also Workable calls this step phone screen too.

File: roles/hiring.md (hunk 11 above)
Comment (zupo) [reply roles/hiring.md]: Yep, specifically @marbeann.

File: roles/hiring.md (hunk 12 above)
Comment (zupo) [reply roles/hiring.md]: we use `get on Zoom` elsewhere

File: roles/hiring.md (hunk 9 above)
Comment (zupo) [reply roles/hiring.md]: :D

**Pull request merged**